test/test_torstate.py

```diff
@@ -29,8 +29,7 @@
 from txtorcon.interface import IStreamListener
 from txtorcon.interface import StreamListenerMixin
 from txtorcon.interface import CircuitListenerMixin
-from txtorcon.torstate import _extract_reason
-from txtorcon.circuit import _get_circuit_attacher
+from txtorcon.circuit import _get_circuit_attacher, extract_reason
 
```

Comment:
I know I'm not *completely* consistent, but I'd rather keep one import per line wherever possible ;)